### PR TITLE
Add in storybook example for StepNav

### PR DESCRIPTION
**Pull request by bradcypert**
I tried to find a reason to add some dynamic knobs and selects to modify this example, but really couldn't think of one. Ultimately, this ended up being a pretty simple one to storybook.

### CODE REVIEW

File: src/js/components/StepNav/StepNav.stories.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import StepNav from "./StepNav";
+
+const stories = storiesOf("StepNav", module);
+
+stories.add("Default", () => (
+  <div style={{ padding: 20 + "px" }}>
```
Comment (dmccraw) [inline src/js/components/StepNav/StepNav.stories.js]: don't know if we still need the padding still with the latest changes from Bill and Sara.

File: src/js/components/StepNav/StepNav.stories.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          name: "Message your coach to let them know you're signed up!",
+          icon: "messages"
+        }
+      ]}
+    />
+  </div>
+));
+
+stories.add("Small", () => (
+  <div style={{ padding: 20 + "px" }}>
```
Comment (dmccraw) [inline src/js/components/StepNav/StepNav.stories.js]: same

File: src/js/components/StepNav/StepNav.stories.js (hunk 1 above)
Comment (bradcypert) [reply src/js/components/StepNav/StepNav.stories.js]: It's still a little tight even with those paddings. :\

File: src/js/components/StepNav/StepNav.stories.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          }
+        },
+        {
+          name: "Message your coach to let them know you're signed up!",
+          icon: "messages"
+        }
+      ]}
+    />
+  </div>
+));
```
Comment (dmccraw) [inline src/js/components/StepNav/StepNav.stories.js]: can you add an eof?

File: src/js/components/StepNav/StepNav.stories.js (hunk 1 above)
Comment (bradcypert) [reply src/js/components/StepNav/StepNav.stories.js]: Screen Shot 2019-06-05 at 10 20 02 AM

File: src/js/components/StepNav/StepNav.stories.js (hunk 3 above)
Comment (bradcypert) [reply src/js/components/StepNav/StepNav.stories.js]: added

**Pull request merged**